src/items/class-features/daring_explorer_(ex).json

```diff
@@ -37,11 +37,49 @@
       "gmnotes": "",
       "short": "",
       "unidentified": "",
-      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.On2ZrPJTe8FURsXY]{Operative Exploit}</p>\n<p>When you fail an Acrobatics or Athletics check, you can spend 1 Resolve Point to either reroll the check and use the new result or add 5 to your result and use the new value to determine whether you succeed at the check. You can’t use this ability on @UUID[Compendium.sfrpg.class-features.MBNI94tbMoqizPUk]{Trick Attack (Ex)} attempts. Once you’ve used this ability, you can’t use it again until you’ve taken a 10-minute rest to recover Stamina Points.</p>"
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.On2ZrPJTe8FURsXY]{Operative Exploit}</p>\n<p>When you fail an @Check[type:acrobatics] or @Check[type:athletics], you can spend 1 Resolve Point to either reroll the check and use the new result or add 5 to your result and use the new value to determine whether you succeed at the check. You can’t use this ability on @UUID[Compendium.sfrpg.class-features.MBNI94tbMoqizPUk]{Trick Attack (Ex)} attempts. Once you’ve used this ability, you can’t use it again until you’ve taken a 10-minute rest to recover Stamina Points.</p>"
```

Comment:
I recommend going over these again and ensuring `@Check` is only used when the ability calls for a check (from you or an enemy), not when it is simply talking about a skill/save/etc